other/upgrade_2-1_mysql.sql

```diff
@@ -2347,6 +2347,10 @@ if ($doChange)
 ALTER TABLE {$db_prefix}log_floodcontrol ADD COLUMN ip VARBINARY(16);
 ---#
 
+---# Modify log_type size
+ALTER TABLE {$db_prefix}log_floodcontrol MODIFY log_type VARCHAR(30) NOT NULL DEFAULT 'post';
+---#
```

Comment:
Need to update the pk constraint.

Reply:
What do you mean?

Reply:
Something like that
```
PRIMARY KEY (ip, log_type)
```
you need to add in the upgrade part of mysql.

Reply:
But its [already there](https://github.com/SimpleMachines/SMF2.1/blob/5131f225f88dacd6bb009e6ea9b022793a028245/other/upgrade_2-1_mysql.sql#L2355)

Reply:
Okay but this differently to the install pk where (30) is defined in pk